links/src/utils/generateLinks.ts

```diff
@@ -1,7 +1,8 @@
+import fs from "fs";
 import { getLinks } from "./getLinks";
 
 type GenerateLinks = {
-	(): Promise<any[]>;
+	(): Promise<any>;
```

Comment:
```suggestion
	(): Promise<any>;
```

Use your type from .test.ts here